tests/pymrcpp/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/test_pymrcpp3d.py ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}/test_pymrcpp3d.py)
```

Comment:
I think it'd be preferable if these tests were added to the test suite, _i.e._ if it were possible to run them through `ctest`. Here is an example: https://github.com/bast/cmake-recipes/tree/master/chapter-09/recipe-05/cxx-example Let me know if you need help to set it up wrt location of the generated binding library and so forth.